tests/src/Kernel/Paragraphs/MediaParagraphsTest.php

```diff
@@ -476,6 +477,10 @@ public function testBanner(): void {
 
     // Variant - image / Modifier - hero_left / Full width - No.
     $paragraph->get('field_oe_banner_type')->setValue('hero_left');
+    $paragraph->set('field_oe_link', [
+      'uri' => 'https://example.com',
+      'title' => 'Example',
+    ]);
```

Comment:
After this we should assert the icon is not present.

Reply:
In fact, the external icon should replace now the internal one after updating the link.